kubernetes/thelounge.yml.erb

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: service
+spec:
+  selector:
+    app: thelounge
+  ports:
+    - port: 80
+      targetPort: 9000
```

Comment:
wait, but the container is exposing port 80. How does this work?

Reply:
The container is exposing 9000.

Reply:
https://github.com/ocf/thelounge/blob/master/config.js#L4

Reply:
I guess the container port 80 does not actually matter? Let me test the port number.